client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -0,0 +1,208 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Sidebar from 'layout/sidebar';
+import SidebarFooter from 'layout/sidebar/footer';
+import SidebarItem from 'layout/sidebar/item';
+import SidebarMenu from 'layout/sidebar/menu';
+import SidebarRegion from 'layout/sidebar/region';
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class JetpackCloudSidebar extends Component {
+	static propTypes = {
+		context: PropTypes.shape( {
+			path: PropTypes.string,
+		} ),
+	};
+
+	state = {
+		expandedSections: [],
```

Comment:
👍 Noting that I'm happy to keep the expanded/collapsed state of sections inside a component state for now. While for my-sites's sidebar we use Redux, I think that's overkill and we won't ever be using that in any other place.

Reply:
This was my reasoning as well. Also, since we don't have a store yet, keeping the state inside the component was the only way to implement expandables.